### ISSUE TITLE
Support for ZenUML diagrams

### ISSUE THREAD

**Issue by jwatt** [mgaitan/sphinxcontrib-mermaid#166]
Could configuration to enable the newer ZenUML support be added?

According to the [Integrating with your library/website](https://mermaid.js.org/syntax/zenuml.html#integrating-with-your-library-website) section of https://mermaid.js.org/syntax/zenuml.html it involves something like this:

```
<script type="module">
  import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
  import zenuml from 'https://cdn.jsdelivr.net/npm/@mermaid-js/mermaid-zenuml@0.1.0/dist/mermaid-zenuml.esm.min.mjs';
  await mermaid.registerExternalDiagrams([zenuml]);
</script>
```

**Comment by jwatt** [comment]
(Similar request to #98.)

**Comment by timkpaine** [comment]
Yes I think this would be quite easy to do. I will see about a refactor as the inline JS has gotten a little complicated, but now that we got 1.0 out perhaps we can do a bit of a cleanup for 1.1

**Comment by jwatt** [comment]
That sounds great. Thank you for the very prompt reply.

**Comment by timkpaine** [comment]
This should be much easier after https://github.com/mgaitan/sphinxcontrib-mermaid/pull/210, just follow the same steps as elk, feel free to open a PR.
